Changed to select the progress log level.

gichan-jang opened the pull request:
~Because the test result is printed after the tests are finished, the test results are not printed in deadlock status.~
~Change to print the test name first and start the test.~

Changed to select the progress log level.

  - 0: deactivate the log
  - 1: print test group name
  
Signed-off-by: gichan-jang <gichan2.jang@samsung.com>

myungjoo commented on ssat.sh:
```diff
@@ -217,6 +217,10 @@ do
+
+	printf "${Green}==================================================${NC}\n"
+	printf "    Test Group ${Green}$CASENAME${NC} Starts.\n"
+
```
If you need this, use ```-p```.
If you need this for each "casename" only, let's define ```-p={$n}```, where n is a number of "log level".

gichan-jang commented:
Oh.. `-p` option is what I want. 
Should I make the log level two?
 -  0: default, print all progress 
 -  1: print testcase name only

myungjoo commented:
> Oh.. `-p` option is what I want.
> Should I make the log level two?
> 
> * 0: default, print all progress
> * 1: print testcase name only

0: the default. (do not print progress)
1: print case name (minimal)
9: print all (the current behavior of -p)

2~8: reserved. (same as 9 until it is defined later)

gichan-jang committed:
Add progress log level

Changed to select the progress log level.

  - 0: deactivate the log
  - 1: print test group name
  - 9: print all logs

Signed-off-by: gichan-jang <gichan2.jang@samsung.com>

gichan-jang commented:
> > Oh.. `-p` option is what I want.
> > Should I make the log level two?
> > 
> > * 0: default, print all progress
> > * 1: print testcase name only
> 
> 0: the default. (do not print progress)
> 1: print case name (minimal)
> 9: print all (the current behavior of -p)
> 
> 2~8: reserved. (same as 9 until it is defined later)

Thanks for the guide. Please check it again.

myungjoo commented on ssat.sh:
```diff
@@ -120,7 +120,12 @@ do
+		printf "    --progress or -p or -p=(0,1,9)\n"
+		printf "        '0'    : Used to deactivate log.\n"
+		printf "        '1'    : Print test group name only.\n"
+		printf "        '9'    : [default] Print all progress logs.\n"
```
```"        '0'    : Used to deactivate log.\n"```
-->
```"        '0'    : Do not print logs in progress. If -p is not given, -p=0 is assumed.\n"```

```"        '1'    : Print test group name only.\n"```
-->
```"        '1'    : Print test group names only in progress.\n"```


```"        '9'    : [default] Print all progress logs.\n"```
-->
```"        '2-9'    : Print all logs in progress. If -p is given without numbers, -p=9 is used.\n"```

gichan-jang replied:
Thanks! Updated.

myungjoo commented on ssat-api.sh:
```diff
@@ -140,7 +140,7 @@ function testInit() {
 ## @param $5 set 1 if $1==0 is success and $1!=0 is fail.
 function testResult() {
-	if [[ "${PROGRESS}" -eq "1" ]]
+	if [[ ${PROGRESSLOGLEVEL} -gt 1 ]]
```
gt --> ge ?
